### PR TITLE
Website: A number of corrections in the docs

### PR DESCRIPTION
**Pull request by aLekSer**
Corrected spelling and some missleading phrases.

### CODE REVIEW

File: site/content/en/docs/Guides/metrics.md [hunk 1]
```diff
@@ -184,7 +184,7 @@ Grafana is a open source time series analytics platform which supports Prometheu
 First we will install [Agones dashboard](#grafana-dashboards) as [config maps](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/) in our cluster.
 
 ```bash
-kubectl apply -f ../build/grafana/
+kubectl apply -f ./build/grafana/
```
Comment (aLekSer) [inline site/content/en/docs/Guides/metrics.md:187]: This change was done to comply with prometheus installation step, so all command are performed from Agones repo root.

File: site/content/en/docs/Getting Started/create-fleetautoscaler.md [hunk 2]
```diff
@@ -260,10 +260,10 @@ If you want to update a `Fleet` which has `RollingUpdate` replacement strategy a
 1. With `kubectl apply`: you should omit `replicas` parameter in a `Fleet` Spec before re-applying the `Fleet` configuration.
 1. With `kubectl edit`: you should not change the `replicas` parameter in the `Fleet` Spec when updating other field parameters.
 
-If you follow the rules above, then `maxSurge` and `maxUnavaiable` parameters will be used as the RollingUpdate strategy updates your Fleet.
-Otherwise the Fleet would be scaled according to Fleet `replicas` parameter first and only after certain amount of time it would be rescaled to fit `FleetAutoscaler` `BufferSize` parameter.
+If you follow the rules above, then `maxSurge` and `maxUnavailable` parameters will be used as the RollingUpdate strategy updates your Fleet.
```
Comment (markmandel) [inline site/content/en/docs/Getting Started/create-fleetautoscaler.md:263]:
```suggestion
If you follow the rules above, then the `maxSurge` and `maxUnavailable` parameters will be used as the RollingUpdate strategy updates your Fleet.
```

File: site/content/en/docs/Getting Started/create-webhook-fleetautoscaler.md [hunk 3]
```diff
@@ -20,8 +20,7 @@ period (currently 30s) with a JSON body, and scale the target fleet based on the
 
 ### Prerequisites
 
-It is assumed that you have read the instructions to [Create a Game Server Fleet]({{< relref "./create-fleet.md" >}})
-and you have a running fleet of game servers or you could run command from Step #1.
+It is assumed that you have read the instructions to [Create a Game Server Fleet]({{< relref "./create-fleet.md" >}}).
```
Comment (markmandel) [inline site/content/en/docs/Getting Started/create-webhook-fleetautoscaler.md:23]:
```suggestion
It is assumed that you have completed the instructions to [Create a Game Server Fleet]({{< relref "./create-fleet.md" >}}) and have a running fleet of game servers.
```

Or am wrong, do you not need the Fleet?

File: site/content/en/docs/Getting Started/edit-first-gameserver-go.md [hunk 4]
```diff
@@ -27,16 +27,16 @@ Also complete the "Enabling creation of RBAC resources" and "Installing Agones"
 ### Modify the simple-udp example source code
 Modify the main.go file. For example:
 
-Change main.go line 92:
+Change following line in `main.go`:
```
Comment (markmandel) [inline site/content/en/docs/Getting Started/edit-first-gameserver-go.md:30]:
```suggestion
Change the following line in `main.go`:
```

File: site/content/en/docs/Getting Started/edit-first-gameserver-go.md [hunk 5]
```diff
@@ -81,7 +81,7 @@ Modify the following line from gameserver.yaml to use the new configuration.
 ```
 
 ### If using GKE, deploy Server to GKE
-Apply the latest settings to kubernetes container.
+Apply the latest settings to Kubernetes container.
```
Comment (markmandel) [inline site/content/en/docs/Getting Started/edit-first-gameserver-go.md:84]:
```suggestion
Apply the latest settings to the Kubernetes container.
```

File: site/content/en/docs/Advanced/scheduling-and-autoscaling.md [hunk 6]
```diff
@@ -40,7 +40,7 @@ and the [Fleet Autoscaler Specification]({{< relref "../Reference/fleetautoscale
 
 ## Autoscaling Concepts
 
-To facilitate autoscaling, we need to combine several piece of concepts and functionality, described below.
+To facilitate autoscaling, we need to combine several pieces of concepts and functionality, described below.
```
[the remark this answers is not in the document]
Comment (aLekSer) [reply site/content/en/docs/Advanced/scheduling-and-autoscaling.md:43]: That sounds better now.

File: site/content/en/docs/Getting Started/create-webhook-fleetautoscaler.md (hunk 3 above)
Comment (aLekSer) [reply site/content/en/docs/Getting Started/create-webhook-fleetautoscaler.md:23]: Here we are creating the fleet in step 1 of this tutorial. So this tutorial a bit different to the `create-fleetautoscaler.md`.

**Pull request merged**